__tests__/Ship.test.js

```diff
@@ -0,0 +1,68 @@
+const Ship = require("../src/Ship");
+const Port = require("../src/Port");
+const Itinerary = require("../src/Itinerary");
```

Comment:
These imports aren't used and should be deleted

Reply:
Deleted imports